docs/docs/references/dbt-write-back.mdx

```diff
@@ -4,9 +4,9 @@ import TablesConfigurationWriteBack from './assets/tables-configuration-write-ba
 
 # dbt Write-Back
 
-dbt Write-Back allows you to create models in dbt from queries you've built in the [SQL Runner](/references/sql-runner). You can then explore these models as Tables in Lightdash. Unlike [Virtual Views](/references/virtual-views), these models will be saved to and managed in your dbt project. 
+dbt Write-Back allows you to create models in dbt from queries you've built in the [SQL Runner](/references/sql-runner). You can then explore these models as Tables in Lightdash. Unlike [Virtual Views](/references/virtual-views), these models will be saved to and managed in your dbt project.
```

Comment:
file not changed, but it automatically applied so formatting after updating roles.mdx